packages/slice-machine/src/models/slice/reducer.ts

```diff
@@ -36,6 +36,7 @@ export function reducer(
       case SliceActions.Push:
         return {
           ...prevState,
+          __status: LibStatus.Synced,
```

Comment:
I'm not sure this is the right move.
In a sense that all the other case don't have a status set.

Reply:
It should be set only once by the `__status: (() => {})` below based on row data.